### PR TITLE
Bounties 0077 js samples 2 Airgapped Wallet

### PR DESCRIPTION
**Pull request by AlexanderBuzz**
Airgapped Wallet example in JavaScript

### CODE REVIEW

**Comment by wojake** [comment]
The implementation seems to be correct but I haven't read the code line by line. Could you please move the [README](https://github.com/XRPLF/xrpl-dev-portal/blob/22b77ebc65d9e90d55deed3d5d87b14943fdbe6b/content/_code-samples/airgapped-wallet/py/README.md) file from the py folder to the main [folder](https://github.com/XRPLF/xrpl-dev-portal/tree/22b77ebc65d9e90d55deed3d5d87b14943fdbe6b/content/_code-samples/airgapped-wallet)?

**Comment by AlexanderBuzz** [comment]
> The implementation seems to be correct but I haven't read the code line by line. Could you please move the [README](https://github.com/XRPLF/xrpl-dev-portal/blob/22b77ebc65d9e90d55deed3d5d87b14943fdbe6b/content/_code-samples/airgapped-wallet/py/README.md) file from the py folder to the main [folder](https://github.com/XRPLF/xrpl-dev-portal/tree/22b77ebc65d9e90d55deed3d5d87b14943fdbe6b/content/_code-samples/airgapped-wallet)?

Done and modified to work with the JavaScript examples

**Comment by AlexanderBuzz** [comment]
Oops, I accidentally closed this one, it's reopened now...

**Comment by jonathanlei** [comment]
@wojake could your provide your review? since you authored the python version

**Review by jonathanlei — APPROVED** [review]

File: content/_code-samples/airgapped-wallet/js/airgapped-wallet.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        'Destination': destination
+    }
+
+    // Normally we would fetch certain needed values like Fee,
+    // LastLedgerSequence snd programmatically, like so:
+    //
+    // const preparedTx = await client.autofill(paymentTx)
+    //
+    // But since this is an airgapped wallet without internet
+    // connection, we have to do it by manually:
```
Comment (wojake) [inline content/_code-samples/airgapped-wallet/js/airgapped-wallet.js]:
```suggestion
    // connection, we have to do it manually:
```

**Comment by wojake** [comment]
Your code uses both `ripple-keypairs/dist/` and `xrpl`, couldn't we just use the `xrpl` library for the functions used in `ripple-keypairs/dist/`?

**Comment by wojake** [comment]
Thanks for submitting this. Other than the nit suggestions that I just pointed out, I don't think there are any odd bugs 😄

**Pull request merged**